framework/customizations/arg_parser.py

```diff
@@ -50,6 +57,7 @@ def process_arguments(arguments):
     # load nsfs_installation and db_installation values to config
     framework.config.ENV_DATA["nsfs_installation"] = nsfs_installation
     framework.config.ENV_DATA["db_installation"] = db_installation
+    framework.config.ENV_DATA["rpm"] = rpm
```

Comment:
Just overwrite: `noobaa_sa`
```suggestion
    framework.config.ENV_DATA["noobaa_sa"] = args.rpm
```
Then we can avoid code here:
deployment/deployment.py

Reply:
done